_overviews/scala3-book/collections-classes.md

```diff
@@ -0,0 +1,600 @@
+---
+title: Collections Classes
+type: section
+description: This page introduces the common Scala 3 collections classes.
+num: 37
+previous-page: collections-intro
+next-page: collections-methods
+---
+
+
+{% comment %}
+TODO: add a correct hierarchy image
+{% endcomment %}
+
```

Comment:
I do not agree with using the term `classes` so much in this file, it is not completely correct.
 `List` could, in the future, be represented as an `enum` and `Map` is only a trait with a companion object that uses subtypes of Map, not a class. The same goes for `Set`

Reply:
Thanks for this. I changed most occurrences to “types,” or otherwise deleted them.